CIFAR/main.swift

```diff
@@ -27,15 +29,27 @@ for epoch in 1...10 {
         trainingBatchCount += 1
         optimizer.update(&model.allDifferentiableVariables, along: gradients)
     }
-    print("  average loss: \(trainingLossSum / Float(trainingBatchCount))")
-    print("Epoch \(epoch), evaluating on test set...")
+
     var testLossSum: Float = 0
     var testBatchCount = 0
+    var correctGuessCount:Int = 0
```

Comment:
Integer literals default to `Int`.

```suggestion
    var correctGuessCount = 0
```

Reply:
removed